compiler/rustc_proc_macro/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+# We need to use a separate crate including library/proc_macro as opposed to a
+# direct path dependency on library/proc_macro because doing the latter will
+# cause two copies of libproc_macro.rlib to end up in the sysroot, breaking
+# proc-macro crates. In addition it confuses the workspace_members function of
+# bootstrap.
+
+[package]
+name = "rustc_proc_macro"
+version = "0.0.0"
+edition = "2024"
+
+[lib]
+path = "../../library/proc_macro/src/lib.rs"
```

Comment:
This will break for the same reason, no? That path just doesn't exist when rustc-dev is distributed via rustup.

Reply:
That didn't work. Building two crates both named proc_macro (once as part of stdlib and once as part of rustc) causes ambiguity for `extern crate proc_macro;` when the rustc libraries are part of the sysroot. Because of this I'm forced to build the rustc copy with a different crate name, which requires a separate Cargo.toml.